`@woocommerce/dependency-extraction-webpack-plugin` version 2.4.0 is missing files

### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The `@woocommerce/dependency-extraction-webpack-plugin` packages on NPM looks to be missing some files.  
It can be seen on the npm repository here:

- v2.4.0 - https://www.npmjs.com/package/@woocommerce/dependency-extraction-webpack-plugin/v/2.4.0?activeTab=code
- v2.3.0 - https://www.npmjs.com/package/@woocommerce/dependency-extraction-webpack-plugin/v/2.3.0?activeTab=code

### Expected behavior

Using the released version of the `@woocommerce/dependency-extraction-webpack-plugin` works correctly.

### Actual behavior

An error is return when the `@woocommerce/dependency-extraction-webpack-plugin` is used in a webpack configuration (example uses wp-scripts):
```
$ wp-scripts build
[webpack-cli] Failed to load '/path/to/project/webpack.config.js' config
[webpack-cli] Error: Cannot find module '../assets/packages'
Require stack:
- /path/to/project/node_modules/@woocommerce/dependency-extraction-webpack-plugin/src/index.js
- /path/to/project/webpack.config.js
```

### Steps to reproduce

1. Create in `package.json`
2. Install `@wordpress/scripts` and `@woocommerce/dependency-extraction-webpack-plugin` as dependencies
3. Create a `webpack.config.js` like:
    ```
    const defaultConfig = require('@wordpress/scripts/config/webpack.config');
    const WooCommerceDependencyExtractionWebpackPlugin = require('@woocommerce/dependency-extraction-webpack-plugin');
    
    module.exports = {
        ...defaultConfig,
        entry: {
            'plugin': './resources/js/plugin',
        },
        output: {
            publicPath: './',
            path: __dirname + '/assets/js',
            filename: '[name].js',
        },
        plugins: [
            ...defaultConfig.plugins.filter(
                (plugin) =>
                    plugin.constructor.name !== 'DependencyExtractionWebpackPlugin'
            ),
            new WooCommerceDependencyExtractionWebpackPlugin()
        ]
    };
```
4. Run `npm install` and `npm run build`
5. Observe error message being returned

### WordPress Environment

```
...
    "devDependencies": {
        "@woocommerce/dependency-extraction-webpack-plugin": "2.4.0",
        "@wordpress/scripts": "26.19.0"
    },
...
```

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

--- a/packages/js/dependency-extraction-webpack-plugin/changelog/46755-dev-add-files-to-dewp-dist
+++ b/packages/js/dependency-extraction-webpack-plugin/changelog/46755-dev-add-files-to-dewp-dist
@@ -1,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix a bug where the assets folder was not distributed with @woocommerce/dependency-extraction-webpack-plugin

--- a/packages/js/dependency-extraction-webpack-plugin/package.json
+++ b/packages/js/dependency-extraction-webpack-plugin/package.json
@@ -22,7 +22,8 @@
 	},
 	"main": "src/index.js",
 	"files": [
-		"src/"
+		"src/",
+		"assets/"
 	],
 	"dependencies": {
 		"@wordpress/dependency-extraction-webpack-plugin": "^3.7.0"

--- a/packages/js/dependency-extraction-webpack-plugin/package.json
+++ b/packages/js/dependency-extraction-webpack-plugin/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@woocommerce/dependency-extraction-webpack-plugin",
-	"version": "2.4.1",
+	"version": "2.4.0",
 	"description": "WooCommerce Dependency Extraction Webpack Plugin",
 	"author": "Automattic",
 	"license": "GPL-2.0-or-later",

--- a/packages/js/dependency-extraction-webpack-plugin/package.json
+++ b/packages/js/dependency-extraction-webpack-plugin/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@woocommerce/dependency-extraction-webpack-plugin",
-	"version": "2.4.0",
+	"version": "2.4.1",
 	"description": "WooCommerce Dependency Extraction Webpack Plugin",
 	"author": "Automattic",
 	"license": "GPL-2.0-or-later",

